BUILD.md

```diff
@@ -104,6 +104,87 @@ The following dependencies are required for this project:
     ctest --test-dir build --output-log build
     ```
 
+## Compilation steps for Windows
+
+1. **Installing Dependencies**
+
+- Microsoft Visual C++ (MSVC) 14.
+- Chocolatey
+    ```bash
+    Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+    ```
+- Make
+    ```bash
+    choco insall make
+    ```
+- Cmake 3.30.3
+    ```bash
+    choco insall cmake --installargs 'ADD_CMAKE_TO_PATH=System'
+    ```
+- LLVM (LLVM-18.1.8-win64.exe) from https://github.com/llvm/llvm-project/releases/tag/llvmorg-18.1.8
```

Comment:
Why is LLVM needed in MSVC?

Reply:
It is needed because clang-tidy and clang-format are part of the LLVM suite on Windows. Without this suite, I couldn't build the project. I will try in a clean VM with this
```
choco install clang-tidy
choco install clang-format
```

Reply:
I think that visual studio already comes with clang format. And although we use clang-tidy for development, I don't see it as necessary to get a build... of course this all makes sense if we make a distinction between building for development and not. I'm not aware if that was discussed and if this file with instructions is meant for development or not.

Reply:
Finally, LLVM was not necessary.